### ISSUE TITLE
Failed to activate the language-archlinux package

### ISSUE THREAD

**Issue by bilelmoussaoui** [fusion809/language-archlinux#12]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.8.0
**System**: "Arch Linux"
**Thrown From**: [language-archlinux](https://github.com/fusion809/language-archlinux) package, v0.5.2
### Stack Trace

Failed to activate the language-archlinux package

```
At Cannot read property 'split' of undefined

TypeError: Cannot read property 'split' of undefined
    at Object.<anonymous> (/home/bilal/.atom/packages/language-archlinux/lib/main.coffee:4:50)
    at Object.<anonymous> (/home/bilal/.atom/packages/language-archlinux/lib/main.coffee:1:1)
    at Module._compile (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (/usr/share/atom/resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (/usr/share/atom/resources/app.asar/src/package.js:715:27)
    at Package.module.exports.Package.activateNow (/usr/share/atom/resources/app.asar/src/package.js:173:16)
    at /usr/share/atom/resources/app.asar/src/package.js:795:25
    at Function.module.exports.Emitter.simpleDispatch (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:241:20)
    at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/usr/share/atom/resources/app.asar/src/window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (/usr/share/atom/resources/app.asar/src/window-event-handler.js:3:61)
```
### Commands

```
     -0:11.8.0 application:new-file (atom-text-editor.editor.is-focused)
     -0:06.4.0 core:save (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "autoHideMenuBar": true,
    "disabledPackages": [
      "terminal-plus",
      "vim-mode",
      "activate-power-mode"
    ],
    "packagesWithKeymapsDisabled": [],
    "themes": [
      "atom-material-ui",
      "solarized-light-syntax"
    ]
  },
  "language-archlinux": {
    "checkPKGBUILDOnSave": true,
    "updateCheckSumsOnSave": true,
    "updateSrcInfoOnSave": true
  }
}
```
### Installed Packages

``` coffee
# User
atom-beautify, v0.29.8
atom-material-ui, v1.2.13
auto-detect-indentation, v1.1.0
autocomplete-python, v1.8.1
color-picker, v2.1.1
emmet, v2.4.3
file-icons, v1.7.15
flex-tool-bar, v0.10.2
gtk-dark-theme, v0.1.4
highlight-line, v0.11.1
highlight-selected, v0.11.2
language-archlinux, v0.5.2
language-arduino, v0.4.1
language-unix-shell, v0.34.2
linter, v1.11.14
minimap, v4.24.6
pigments, v0.29.2
python-autopep8, v0.1.3
python-isort, v0.0.7
tool-bar, v0.4.0
tool-bar-almighty, v0.6.1

# Dev
No dev packages
```

**Comment by fusion809** [comment]
I have attempted to fix this in commit a754143. Please try it, by re-installing `language-archlinux` with:

``` bash
apm remove language-archlinux
apm install fusion809/language-archlinux
```

**Comment by fusion809** [comment]
Oh btw, have you seen the [Contributors](https://github.com/fusion809/language-archlinux#contributors) section of the README? You are an important contributor to this project, so I'd like to say thank you for that.

**Comment by bilelmoussaoui** [comment]
I will try it soon. Yeah i did, thanks !😃

**Comment by fusion809** [comment]
I experienced this issue to once, so I fixed it in the v0.5.3 release, so I am closing this issue, feel free to open if my solution has failed.

**Comment by bilelmoussaoui** [comment]
Thanks! i didn't update/create any new archlinux package at the moment. Will report back if anything happens.
